cluster/images/hyperkube/kube-proxy.json

```diff
@@ -13,7 +13,8 @@
                 "proxy",
                 "--master=http://127.0.0.1:8080",
                 "--v=2",
-                "--resource-container=\"\""
+                "--resource-container=\"\"",
+                "--proxy-mode=iptables"
```

Comment:
I may remove it if you like, it should be default anyway
Ref: 
> --proxy-mode="": Which proxy mode to use: 'userspace' (older) or 'iptables' (faster). If blank, look at the Node object on the Kubernetes API and respect the 'net.experimental.kubernetes.io/proxy-mode' annotation if provided.  Otherwise use the best-available proxy (currently iptables).  If the iptables proxy is selected, regardless of how, but the system's kernel or iptables versions are insufficient, this always falls back to the userspace proxy.